README.md

```diff
@@ -41,7 +41,11 @@ const port = 3000;
 
 const rootPath = path.resolve(__dirname);
 
-app.use(esModuleDevserver.middleware(rootPath));
+app.use(
+    esModuleMiddleware.middleware({
```

Comment:
I wanted to avoid having to change configurations in the future if more options were added. I agree though we should have a unified config, so I've changed the connect-style middleware to share the same implementation.